### PR TITLE
[FLINK-23242][docs-zh] Translate the page of "Handling Application Parameters" into Chinese

### PR DESCRIPTION
**Pull request by zuoniduimian**
What is the purpose of the change
Translate the page of "Handling Application Parameters " into Chinese

The page url of "Handling Application Parameters" is https://ci.apache.org/projects/flink/flink-docs-release-1.13/zh/docs/dev/datastream/application_parameters/.

The markdown file can be found in flink/docs/content.zh/docs/dev/datastream/application_parameters.md in English.

Brief change log
Translate 'flink/docs/content.zh/docs/dev/datastream/application_parameters.md'.
Verifying this change
This change is a trivial rework / code cleanup without any test coverage.

Does this pull request potentially affect one of the following parts:
Dependencies (does it add or upgrade a dependency): no
The public API, i.e., is any changed class annotated with @public(Evolving): no
The serializers: no
The runtime per-record code paths (performance sensitive): no
Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
The S3 file system connector: no
Documentation
Does this pull request introduce a new feature? no
If yes, how is the feature documented? not documented

### CODE REVIEW

**Comment by zuoniduimian** [comment]
@PatrickRen   hello,Could you help me review my PR in your spare time? thank you !

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 1]
```diff
@@ -24,28 +24,23 @@ specific language governing permissions and limitations
...
 
-Flink provides a simple utility called `ParameterTool` to provide some basic tooling for solving these problems.
-Please note that you don't have to use the `ParameterTool` described here. Other frameworks such as [Commons CLI](https://commons.apache.org/proper/commons-cli/) and
-[argparse4j](http://argparse4j.sourceforge.net/) also work well with Flink.
+为解决以上问题，Flink 提供一个名为 `Parametertool` 的简单公共类，其中包含了一些基本的工具。请注意，这里说的 `Parametertool` 并不是必须使用的。[Commons CLI](https://commons.apache.org/proper/commons-cli/) 和 [argparse4j](http://argparse4j.sourceforge.net/) 等其他框架也可以非常好地兼容 Flink。
 
+### `ParameterTool` 读取配置值
 
-### Getting your configuration values into the `ParameterTool`
+`ParameterTool` 已经定义好了一组静态方法，用于读取配置信息。该工具类内部使用了 `Map<string，string>` 类型，这样使得它可以很容易地与你的配置集成在一起。
```
Comment (PatrickRen) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:37]: `ParameterTool` 定义了一组静态方法用于读取配置。

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 2]
```diff
@@ -24,28 +24,23 @@ specific language governing permissions and limitations
...
 
-The `ParameterTool` provides a set of predefined static methods for reading the configuration. The tool is internally expecting a `Map<String, String>`, so it's very easy to integrate it with your own configuration style.
 
+#### 配置值来自 `.properties` 文件
```
Comment (PatrickRen) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:40]: `.properties` 文件中的配置

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 3]
```diff
@@ -57,33 +52,33 @@ InputStream propertiesFileInputStream = new FileInputStream(file);
 ParameterTool parameter = ParameterTool.fromPropertiesFile(propertiesFileInputStream);
 ```
 
+#### 配置值来自命令行
```
Comment (PatrickRen) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:55]: 命令行参数中的配置

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 4]
```diff
@@ -57,33 +52,33 @@ InputStream propertiesFileInputStream = new FileInputStream(file);
...
-This allows getting arguments like `--input hdfs:///mydata --elements 42` from the command line.
 ```java
 public static void main(String[] args) {
     ParameterTool parameter = ParameterTool.fromArgs(args);
     // .. regular code ..
 ```
 
 
-#### From system properties
+#### 配置值来自系统属性
```
Comment (PatrickRen) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:66]: 系统属性中的配置

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 5]
```diff
@@ -93,29 +88,28 @@ parameter.getNumberOfParameters()
...
+然后在函数内使用它以获取命令行的传递的参数。
 
-#### Register the parameters globally
+#### 全局注册参数
 
-Parameters registered as global job parameters in the `ExecutionConfig` can be accessed as configuration values from the JobManager web interface and in all functions defined by the user.
+从 JobManager web 界面和用户定义的所有函数中可以以配置值的方式访问在 `ExecutionConfig` 中注册的全局作业参数。
 
-Register the parameters globally:
+在全局注册参数：
```
Comment (PatrickRen) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:112]: 注册全局参数：

**Review by PatrickRen — CHANGES_REQUESTED** [review]
Thanks for the contribution @zuoniduimian ! The translation overall looks good to me. I left some comments. 

By the way I don't have the permission to merge the PR, but I can help to review and approve. Once you think the PR is ready, please ping @wuchong for the final review and merge the PR for you.

**Comment by zuoniduimian** [comment]
@PatrickRen  Thank you for your valuable comments. I have made some modifications according to your comments.

**Comment by zuoniduimian** [comment]
@wuchong   hello,Could you help me review my PR in your spare time？if there is some problem,please tell me!Thank you!

**Comment by zuoniduimian** [comment]
@wuchong  hello,Jark,if you have some time,could you help me view my translation for the page of "Handling Application Parameters " . Thank you a lot !

**Comment by wuchong** [comment]
@PatrickRen , do you have other concerns about this PR?

**Comment by zuoniduimian** [comment]
@PatrickRen @wuchong  hello，Thank you for your review and comments. Do you think there is anything else that can be modified in my PR?

File: docs/content.zh/docs/dev/datastream/application_parameters.md (hunk 2 above)
Comment (PatrickRen) [reply docs/content.zh/docs/dev/datastream/application_parameters.md:40]: 我觉得 ”配置值来自 `.properties` 文件“ 的语序读起来很奇怪（翻译腔），“来自 `.properties` 文件的配置” 或 “`.properties` 文件中的配置“ 似乎会自然一些

File: docs/content.zh/docs/dev/datastream/application_parameters.md (hunk 3 above)
Comment (PatrickRen) [reply docs/content.zh/docs/dev/datastream/application_parameters.md:55]: 同上述语序的问题

File: docs/content.zh/docs/dev/datastream/application_parameters.md (hunk 4 above)
Comment (PatrickRen) [reply docs/content.zh/docs/dev/datastream/application_parameters.md:66]: 同上述语序的问题

**Review by PatrickRen — CHANGES_REQUESTED** [review]
Thanks for the update @zuoniduimian ~I left some comments about the order of words.

**Comment by zuoniduimian** [comment]
> Thanks for the update @zuoniduimian ~I left some comments about the order of words.

@PatrickRen   @wuchong   With respect to the original translation and contextual coherence, I think this word order is acceptable. However, if both of you feel that the word order needs to be adjusted, I can modify it.

**Comment by PatrickRen** [comment]
According to the [translation specifications](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications), it would be better to use free translation instead of literal (意译优于直译) if the literal translation is hard to understand or against the habit of Chinese. 

I think the order of words is acceptable, just a suggestion 😄

**Comment by zuoniduimian** [comment]
> According to the [translation specifications](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications), it would be better to use free translation instead of literal (意译优于直译) if the literal translation is hard to understand or against the habit of Chinese.
> 
> I think the order of words is acceptable, just a suggestion 😄

@PatrickRen  Thank you very much for your comments and review.  @wuchong  Where else do you think can be modified?

File: docs/content.zh/docs/dev/datastream/application_parameters.md [hunk 6]
```diff
@@ -24,28 +24,23 @@ specific language governing permissions and limitations
...
-Almost all Flink applications, both batch and streaming, rely on external configuration parameters.
-They are used to specify input and output sources (like paths or addresses), system parameters (parallelism, runtime configuration), and application specific parameters (typically used within user functions).
+几乎所有的批和流的 Flink 应用程序，都依赖于外部配置参数。这些配置参数可以用于指定输入和输出源（如路径或地址）、系统参数（并行度，运行时配置）和特定的应用程序参数（通常使用在用户自定义函数）。
 
-Flink provides a simple utility called `ParameterTool` to provide some basic tooling for solving these problems.
-Please note that you don't have to use the `ParameterTool` described here. Other frameworks such as [Commons CLI](https://commons.apache.org/proper/commons-cli/) and
-[argparse4j](http://argparse4j.sourceforge.net/) also work well with Flink.
+为解决以上问题，Flink 提供一个名为 `Parametertool` 的简单公共类，其中包含了一些基本的工具。请注意，这里说的 `Parametertool` 并不是必须使用的。[Commons CLI](https://commons.apache.org/proper/commons-cli/) 和 [argparse4j](http://argparse4j.sourceforge.net/) 等其他框架也可以非常好地兼容 Flink。
 
+### `ParameterTool` 读取配置值
```
Comment (wuchong) [inline docs/content.zh/docs/dev/datastream/application_parameters.md:35]: 用 `ParameterTool` 读取配置值 ?

Comment (zuoniduimian) [reply docs/content.zh/docs/dev/datastream/application_parameters.md:35]: OK, the translation is acceptable and more reasonable

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**